vm.go

```diff
@@ -225,6 +261,23 @@ func (vm *VM) EvaluateSnippetMulti(filename string, snippet string) (files map[s
 	return
 }
 
+// EvaluateSnippetDependencies evaluates a string containing Jsonnet code returning an array of dependencies.
+//
+// The filename parameter is only used for error messages.
+func (vm *VM) EvaluateSnippetDependencies(filename string, snippet string) (dependencies []string, formattedErr error) {
```

Comment:
Same thing about this name – it doesn't really evaluate Jsonnet code. Maybe just FindDependencies?

Reply:
Changed to `FindDependencies`.